cerberus-dashboard/src/actions/authenticationActions.js

```diff
@@ -245,7 +245,44 @@ export function triggerCodeChallenge(mfaDeviceId, stateToken) {
                         </div>
                     </div>
                 ));
+                dispatch(resetAuthState());
+            });
+    };
+}
+
+/**
+ * This action triggers a push notification challenge for a user with MFA, if MFA is required
+ * @param mfaDeviceId ID of the MFA security device
+ * @param stateToken Identifying token for the authentication request
+ */
+export function triggerPushChallenge(mfaDeviceId, stateToken) {
+    return function (dispatch) {
+        dispatch(loginMfaChallenge());
+        return axios({
+            method: 'post',
+            url: environmentService.getDomain() + cms.USER_AUTH_MFA_PATH,
+            data: {
+                state_token: stateToken,
+                device_id: mfaDeviceId,
+                is_push: true
+            },
+            timeout: AUTH_ACTION_TIMEOUT
```

Comment:
This is 60s. Is that standard for push notifications? Does that give users enough time?

Reply:
I kept the same timeout we use for OTP, SMS, and phone. In my experience, 60s should be more than enough time. I haven't timed it but I feel like push MFA is the fastest of the options (no need to find and open an app, wait for a text, or wait for a phone call, and then type the code ). I don't know if there is a standard amount but my thought was if it's long enough for the other factor types it should be long enough for push style.

Reply:
sounds reasonable to me